src/Page/Form.purs

```diff
@@ -100,21 +102,24 @@ component =
       }
 
   render :: State -> H.ParentHTML Query (ChildQuery m) ChildSlot m
-  render { form } =
-    Layout.section_
-      case form of
-        NotAsked ->
-          []
-        Loading ->
-          [ Card.card_ [ Format.p [ css "text-red" ] [ HH.text "Loading..." ] ] ]
-        Failure e ->
-          [ Card.card_ [ Format.p [ css "text-red" ] [ renderEvalError e ] ] ]
-        Success page ->
-          append
-          [ Format.heading_
-            [ HH.text page.evaled.name ]
-          ]
-          $ renderSection <$> page.evaled.contents
+  render { form, values } =
+    HH.div_
+      [ Layout.section_
+        case form of
+          NotAsked ->
+            []
+          Loading ->
+            [ Card.card_ [ Format.p [ css "text-red" ] [ HH.text "Loading..." ] ] ]
+          Failure e ->
+            [ Card.card_ [ Format.p [ css "text-red" ] [ renderEvalError e ] ] ]
+          Success page ->
+            append
+            [ Format.heading_
+              [ HH.text page.evaled.name ]
+            ]
+            $ renderSection <$> page.evaled.contents
+      , HH.pre_ [ HH.text $ show values ]
```

Comment:
Added a little debug feature here, rendering out the current form values so that we can be sure our form's in the correct state. Would be nice if we could pretty print this

Reply:
Ah, I see. So we want to keep this? I'm game. I wonder if there's something in halogen or a halogen package to do this for us?

Reply:
For now, I think it's useful. Eventually we'll want to hide it, maybe expose it via an env flag or a `?debug` param in the url like we do for our current forms